### DISCUSSION TITLE
Unofficial BETA-5 Download

### DISCUSSION THREAD

**Discussion by vicos59** [geodesicsolutions-community/geocore-community#166] [General]
The latest built release available in this repository as of this date is Beta-4. However, Beta-4 is way behind the actual code changes that have been since implemented. 

OPTION 1:
=======

Since the guy who maintains this repo has not been around in quite some time, I went ahead and built what I call an unofficial Beta-5. You can download it from my copy of the GeoCore repo:

https://github.com/vicos59/geocore-community/tree/main/RELEASES/Beta-5

OPTION 2:
=======

If you prefer, you can just download the latest source from this repo:

https://github.com/geodesicsolutions-community/geocore-community

But, I believe you will have to run a composer command or two to get it to pull in all of the required 3rd party code.

OPTION 3:
=======

You can just fork the main repo to your own github account and then run the command to build a new release as described here::

https://github.com/geodesicsolutions-community/geocore-community/discussions/153

How close is Unofficial Beta-5 to be release quality?
=======

Here is what I know:

1. Based on my testing, there are only a few minor bugs and these are mostly in addOn products that few people would ever use. You can follow this thread if you want to see everything that was fixed after Beta-4 until Beta-5: https://github.com/geodesicsolutions-community/geocore-community/discussions/95
2. Testing was done using PHP 7.4 and 8.1
3. Based on my testing, Social Connect (Facebook Login) does not work. FB has changed their methods many times over the past few years. Looks like quite the effort to update it. Be better replacing it with a modern library that allows connection to all modern social media platforms.
4. Payment gateways have not been tested individually. That is a large task. I guess it will be left up to individual users to report back whether or not their gateway of choice works.

**Comment by jonyo** [comment]
> Since the guy who maintains this repo has not been around in quite some time

Hey!  Ok I guess that is fair... In my defense, I did recently get a new toy to shift my "free time" to...  And I must say, beat saber is very fun and addictive!  😂  (Also good exercise!)

I'll see if I can wrap up any of the remaining critical bugs then build an official beta 5 when I get enough time, don't hold me to it, but maybe in 1-2 weeks (remember I do this in spare time and don't make money on it).

**Comment by jonyo** [comment]
By the way:

> You can just fork the main repo to your own github account and then run the command to build a new release as described here::

You don't have to fork it, unless you plan to make changes and you wanted to make a PR.  If you are just building it, you can just clone this repo directly.
